<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminSignUpFormClass</class>
 <widget class="QMainWindow" name="AdminSignUpFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminSignUpForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
				QMainWindow {
				background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #ffffff, stop:1 #a4bedf);
				}
			</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">
					QMainWindow {
					background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0,
					stop:0 #ffffff, stop:0.5 #d3d3d3, stop:1 #d3d3d3);}
				</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>551</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_1">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mật khẩu quản trị viên</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordInput">
       <property name="styleSheet">
        <string notr="true">
									QLineEdit {
									height: 30px;
									border-radius: 0;
									border: 1px solid #999;
									}
								</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Xác nhận mật khẩu</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="confirmedPasswordInput">
       <property name="styleSheet">
        <string notr="true">
									QLineEdit {
									height: 30px;
									border-radius: 0;
									border: 1px solid #999;
									}
								</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="emailInput">
       <property name="styleSheet">
        <string notr="true">
									QLineEdit {
									height: 30px;
									border-radius: 0;
									border: 1px solid #999;
									}
								</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="signUpButton">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
											QPushButton {
											height: 30px;
											width: 90px;
											color: #ffffff;
											background-color: #2681c9;
											border-radius: 5px;
											border: 2px solid #99c6e9;
											cursor: pointer;
											}

											QPushButton::hover {
											background-color: #4a9fd1;
											}
										</string>
         </property>
         <property name="text">
          <string> OK</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Images/checkmark.png</normaloff>../../Images/checkmark.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
											QPushButton {
											height: 30px;
											width: 90px;
											color: #ffffff;
											background-color: #2681c9;
											border-radius: 5px;
											border: 2px solid #99c6e9;
											cursor: pointer;
											}

											QPushButton::hover {
											background-color: #4a9fd1;
											}
										</string>
         </property>
         <property name="text">
          <string> Hủy</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Images/close.png</normaloff>../../Images/close.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
						QLabel {
						font: 9pt;
						text-align: left;
						}
					</string>
    </property>
    <property name="text">
     <string>Đây là lần đầu tiên bạn chạy IriTracker trên máy tính này. Trước khi tiếp tục, bạn phải nhập mật khẩu quản trị viên và email để nhắc mật khẩu. Mật khẩu phải dài ít nhất 4 ký tự.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
